money.go

```diff
@@ -112,7 +117,11 @@ func NewFromString(amount string, currencyCode string) (*Money, error) {
 
 	toParse := amount
 	var decimals int
-	if pointIndex := strings.Index(amount, currency.Decimal); pointIndex != -1 {
+	// There are some cases where decimal are interpreted with "," sign.
+	// since we are using decimal numbers like 1.2
+	// we are not gonna use currency decimal sign as provided from currency here.
+	// (To avoid changes in currencies itself and to still offer Format function)
+	if pointIndex := strings.Index(amount, "."); pointIndex != -1 {
```

Comment:
I see that you are doing this because we always represent the money in "XXXXXX.XX" format no matter what currency we talk about. I am fine with this, my only concern is that we are deviating from the original go-money which means that it will be harder to maintain. Given that `NewFromString` is something that they don't have yet I don't see a big problem though.